packages/apps/reputation-oracle/server/src/database/migrations/1731562977435-AddWebhookOutgoingAndEscrowCompletionTrackingTables.ts

```diff
@@ -0,0 +1,99 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddWebhookOutgoingAndEscrowCompletionTrackingTables1731562977435
+  implements MigrationInterface
+{
+  name = 'AddWebhookOutgoingAndEscrowCompletionTrackingTables1731562977435';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "hmt"."webhook_incoming" DROP COLUMN "results_url"`,
+    );
+    await queryRunner.query(
+      `ALTER TABLE "hmt"."webhook_incoming" ADD COLUMN "failure_detail" varchar`,
+    );
+    await queryRunner.query(
+      `CREATE TYPE "hmt"."webhook_outgoing_status_enum" AS ENUM('pending', 'sent', 'failed')`,
+    );
+    await queryRunner.query(
+      `CREATE TABLE "hmt"."webhook_outgoing" ("id" SERIAL NOT NULL, "created_at" TIMESTAMP WITH TIME ZONE NOT NULL, "updated_at" TIMESTAMP WITH TIME ZONE NOT NULL, "payload" jsonb NOT NULL, "hash" character varying NOT NULL, "url" character varying NOT NULL, "failure_detail" character varying, "retries_count" integer NOT NULL, "wait_until" TIMESTAMP WITH TIME ZONE NOT NULL, "status" "hmt"."webhook_outgoing_status_enum" NOT NULL, CONSTRAINT "PK_d232b33ccee326e251936e16c5f" PRIMARY KEY ("id"))`,
+    );
+    await queryRunner.query(
+      `CREATE UNIQUE INDEX "IDX_d33daa25634fccb127ead889d4" ON "hmt"."webhook_outgoing" ("hash") `,
+    );
+    await queryRunner.query(
+      `CREATE TYPE "hmt"."escrow_completion_tracking_status_enum" AS ENUM('pending', 'paid', 'completed', 'failed')`,
+    );
+    await queryRunner.query(
+      `CREATE TABLE "hmt"."escrow_completion_tracking" ("id" SERIAL NOT NULL, "created_at" TIMESTAMP WITH TIME ZONE NOT NULL, "updated_at" TIMESTAMP WITH TIME ZONE NOT NULL, "chain_id" integer NOT NULL, "escrow_address" character varying NOT NULL, "final_results_url" character varying, "final_results_hash" character varying, "failure_detail" character varying, "retries_count" integer NOT NULL, "wait_until" TIMESTAMP WITH TIME ZONE NOT NULL, "status" "hmt"."escrow_completion_tracking_status_enum" NOT NULL, CONSTRAINT "PK_a6abebca3ce8e49155aaf14ccc8" PRIMARY KEY ("id"))`,
+    );
+    await queryRunner.query(
```

Comment:
Maybe in order to improve index we swap `chain_id` and `escrow_address`? Having `escrow_address` will provide index with better cardinality and hence better performance